tools/lightning-checkpoints.md

```diff
@@ -0,0 +1,92 @@
+---
+title: TiDB Lightning 断点续传
+category: tools
+---
+
+# TiDB Lightning 断点续传
```

Comment:
```suggestion
# TiDB-Lightning 断点续传
```

According to https://github.com/pingcap/tidb-lightning/pull/80#discussion_r232165326 we should hyphenate it